Assets/HoloToolkit-Examples/Throwing/Scripts/ThrowingExampleSceneManager.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.WSA.Input;
+
+public class ThrowingExampleSceneManager : MonoBehaviour
+{
+    public GameObject[] ThrowingAssets;
+    public InteractionSourceNode ControllerPose = InteractionSourceNode.Grip;
+    public Transform RealWorldRoot;
+
+    private readonly Dictionary<uint, Transform> devices = new Dictionary<uint, Transform>();
+    private readonly Dictionary<uint, int> modelIndecies = new Dictionary<uint, int>();
+    private readonly Dictionary<uint, bool> isDetatched = new Dictionary<uint, bool>();
+
+    // Use this for initialization
+    void Start ()
```

Comment:
Missing explicit private access modifiers for unity api's and formatting is off.
We can probably lose the comment too.